recipes/unidock/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "unidock" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dptech-corp/Uni-Dock/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d8147e596079632221ef3c692cb1f1e6fc6e9a1e9303d2bcd44afe1508557af0
+
+build:
+  skip: true  # [not linux]
+  skip: true  # [cuda_compiler_version == "None"]
+  skip: true  # [cuda_compiler_version == "11.2"]
+  script: |
+    cd unidock
+    mkdir conda_build && cd conda_build
+    cmake ${CMAKE_ARGS} ${SRC_DIR}/unidock
+    make unidock -j$CPU_COUNT
+    make install
+  string: cuda{{ cuda_compiler_version | replace('.', '') }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - make
```

It looks like GNU OpenMP is checked for in the linkages. Would add it here

```suggestion
    - make
    - libgomp
```